modules/xfem/src/base/XFEMGeometricCut3D.C

```diff
@@ -0,0 +1,168 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "XFEMGeometricCut3D.h"
+
+#include "MooseError.h"
+#include "XFEMFuncs.h"
+
+XFEMGeometricCut3D::XFEMGeometricCut3D(Real t0, Real t1):
+  XFEMGeometricCut(t0,t1),
+  _center(Point(0.0,0.0,0.0)),
```

Comment:
The default constructor for `Point` initializes the member as null. so this should be `_center(),`!